nomad/grapl-core.nomad

```diff
@@ -797,16 +823,18 @@ job "grapl-core" {
       driver = "docker"
 
       config {
-        image        = "${var.container_registry}grapl/analyzer-dispatcher:${var.analyzer_dispatcher_tag}"
-        network_mode = "grapl-network"
+        image = "${var.container_registry}grapl/${var.container_repo}analyzer-dispatcher:${var.analyzer_dispatcher_tag}"
+      }
+
+      template {
+        data        = local.local_vars
+        destination = "analyzer-dispatcher.env"
+        env         = true
       }
 
       env {
         # AWS vars
-        AWS_REGION                  = var.aws_region
-        GRAPL_AWS_ACCESS_KEY_ID     = var.aws_access_key_id
-        GRAPL_AWS_ACCESS_KEY_SECRET = var.aws_access_key_secret
-        GRAPL_AWS_ENDPOINT          = local.aws_endpoint
+        AWS_REGION = var.aws_region
```

Comment:
We should adopt `AWS_DEFAULT_REGION` here, as pointed out above.

Otherwise, someone reading all these services is going to start asking questions about why some are one way and others are a different way.